server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -89,6 +90,7 @@ pub enum RequisitionRowApprovalStatus {
 pub struct RequisitionRow {
     pub id: String,
     pub requisition_number: i64,
+    #[column_name = "name_link_id"]
```

Comment:
@Chris-Petty is this correct?  i.e. everywhere where the name_id was used the name_link_id is used now?

Reply:
Yup - I think we discussed this IRL now?